packages/expo-dev-menu/android/src/main/java/expo/modules/devmenu/DevMenuReactRootViewContainer.kt

```diff
@@ -12,11 +12,6 @@ import androidx.core.view.doOnLayout
 import com.facebook.react.ReactRootView
 import expo.modules.devmenu.fab.MovableFloatingActionButton
 
-/**
- * We want to hide the FAB behind the feature flag for now.
- */
-private const val enableFAB = false
```

Comment:
perhaps just disabling it by default is sufficient? we could mark as experimental in the launcher ui also